pkg/cli/verifier/testdata/httpbin2_permissive.json

```diff
@@ -0,0 +1,1850 @@
+{
```

Comment:
Is this being used anywhere?

Reply:
Not yet, but added this so all the dumps are from the same verification run. This will be used for the traffic splitting test, which is pending destination side verification.